docs/admin/administration.md

```diff
@@ -344,6 +343,30 @@ boro-11
         Device Reliability: OK
         Read Only: OK
         Volatile Memory Backup: OK
+      Intel Vendor SMART Attributes:
+        Program Fail Count:
+           Normalized(%%):100
```

Comment:
the "%%" output is a mistake - I thought I fixed this already but clearly not. I will file a ticket to address